packages/app/src/bootstrap.tsx

```diff
@@ -1,3 +1,4 @@
+import 'material-icons/iconfont/material-icons.css';
```

Comment:
I don't think we want include icons this way. We should avoid global CSS variables.

Reply:
Hi @jerolimov , if we don't include iconfont, how can the user add a menu group icon when it's not exported in any plugins? So far icons are working because they are being exported in each plugin package.

Reply:
By "generic icons" do you mean icons already exist or available in the app? If the user define a parent menu item, they need to find an existing icon to use? Does it mean we need to let them know which icons are available?

Or, you just mean we define a generic icon, and when user specify an icon doesn't exist in our app we just use the prepared the generic icon for any parent menu items? This way it could be duplicated top level menu icons.